rtgui/cropwindow.cc

```diff
@@ -272,210 +269,209 @@ void CropWindow::buttonPress (int button, int type, int bstate, int x, int y)
 {
 
     bool needRedraw = true;  // most common case ; not redrawing are exceptions
+    EditSubscriber *editSubscriber = iarea->getCurrSubscriber();
 
     iarea->grabFocus (this);
 
-    if (button == 1 && type == GDK_2BUTTON_PRESS && onArea (CropImage, x, y) && (state == SNormal || state == SCropImgMove)) {
-        if (fitZoomEnabled) {
-            if (fitZoom) {
-                state = SNormal;
-                zoomVersion = exposeVersion;
-                screenCoordToImage (x, y, action_x, action_y);
-                changeZoom (ZOOM11INDEX, true, action_x, action_y);
-                fitZoom = false;
+    if (button == 1) {
```

Comment:
You're probably right, I spent quite some time on this function, but it is actually working and because of the deadline, would like to commit as is, unless you want me to change that now.

Please note that we have to check for the mouse button, the area, the modifier keys and the current tool mode. Maybe I could rewrite this function so it is sorted by tool mode first, then area, then mouse buttons, then modifier key ? The actual implementation does handle the priority in a very specific manner.